appinventor/components/src/com/google/appinventor/components/runtime/Clock.java

```diff
@@ -42,6 +42,7 @@
     implements Component, AlarmHandler, OnStopListener, OnResumeListener, OnDestroyListener,
                Deleteable {
   private static final int DEFAULT_INTERVAL = 1000;  // ms
+  private static final boolean DEFAULT_PRECISE_INTERVAL = true;
```

Comment:
I would recommend getting rid of this and just hardcoding the default. I say this because having this definition leads one to think that you can change the default by just changing this value. However changing the defaults has implications about how to handle different (old and new) versions of the Companion app (because default values are not emitted in the Yail).

Reply:
Okay.. but we got a upgrader of properties, it should works

Reply:
Upgraders deal with upgrading projects. I'm talking about dealing with someone using a new version of App Inventor with and older Companion App.